src/client/cli/cmd/alias.cpp

```diff
@@ -110,6 +111,37 @@ mp::ParseCode cmd::Alias::parse_args(mp::ArgParser* parser)
     }
 
     auto instance = definition.left(colon_pos).toStdString();
+    bool instance_exists{false};
+
+    list_request.set_verbosity_level(0);
+    list_request.set_request_ipv4(false);
+
+    auto on_success = [&instance, &instance_exists](ListReply& reply)
+    {
+        for (const auto& reply_instance : reply.instances())
+            if (instance == reply_instance.name())
+            {
+                instance_exists = true;
+                break;
+            }
+
+        return ReturnCode::Ok;
+    };
+
+    auto on_failure = [this](grpc::Status& status) { return ReturnCode::CommandLineError; };
+
+    if (dispatch(&RpcMethod::list, list_request, on_success, on_failure) == ReturnCode::CommandLineError)
```

Comment:
Instead of using the `list` method and then iterating over all of the entries returned, it might be better to use the `info` method on the given instance name and see if returns successfully.

Reply:
Good thought, done!